src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1763,6 +1764,11 @@ public function searchProductsAction(Request $request): JsonResponse
             return $this->json([
                 'products' => $foundProducts,
             ]);
+        } catch (ProductSearchEmptyPhraseException $e) {
+            return $this->json(
+                [$e, 'message' => $this->getErrorMessageForException($e, [])],
```

Comment:
```suggestion
                [$e, 'message' => $this->getErrorMessageForException($e, $this->getErrorMessages($e))],
```

Even if it's not handled for now it will allow adding a custom message for this exception (I guess it was forgotten in the other catch block

Reply:
@jolelievre Fixed